go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/libp2p/go-doh-resolver
+
+go 1.15
+
+require (
+	github.com/ipfs/go-log v1.0.5
```

Comment:
We should probably use go-log v2, right?

Reply:
ah yes, we've switched to v2 pretty much universally; will fix.

Reply:
done.